src/cli/parse-flags.ts

```diff
@@ -201,87 +320,41 @@ const parseStringNumberArg = (flags: ConfigFlags, args: string[], configCaseName
  */
 const CLI_ARG_STRING_REGEX = /[^\d\.Ee\+\-]+/g;
 
+export const Empty = Symbol('Empty');
```

Comment:
I think I see a few advantages in it over just using `string | null` or something. For one, the value at runtime is not falsy, so a missed `=` in a check like `if (value == Empty)` won't ever return false. It also explicitly conveys intent, like if I intend to convey an empty value I _should_ be using something which is distinguished at the type level from a non-empty value — put another way, using `""` to signal an empty string value is ambiguous, because we don't have any way to distinguish between an explicit input of an empty string and an actually missing value. If we use something like this symbol then we can keep an actually fully missing value separate, at both the type- and runtime-level, from any kind of existing value, whether it's an empty string or `"foo"` or `"/path/to/file.foobar"` or whatever.

Does that all make sense?

This is basically me just missing having access to `Option` which exists in Rust (and is very similar to `Maybe` from Haskell and other ML family languages)